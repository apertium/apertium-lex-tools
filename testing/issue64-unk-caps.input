^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^mer<n><pl>/mar<n><sg>$
^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^fleuve<n><pl>/flavor<n>$
^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^London<np><top><sg>/London<np>$
^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*mer/*mer$
^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*M/*M$
^bassin<n><sg>/bloop<n>/blah<n>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*Mn/*Mn$
