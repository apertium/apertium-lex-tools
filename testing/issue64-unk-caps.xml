<rules>
	<rule weight="1.0">
		<match lemma="bassin" tags="n.*"><select lemma="conca"/></match>
		<match lemma="de" tags="pr"/>
		<match tags="det.*"/>
		<or>
			<match lemma="mer" tags="n.*"/>
			<match lemma="fleuve" tags="n.*"/>
			<match lemma="riviÃ¨re" tags="n.*"/>
			<match tags="np.top.*"/>
			<match tags="" case="Aa"/>
		</or>
	</rule>
	<rule weight="0.1">
		<match lemma="bassin" tags="n.*">
			<select lemma="blah"/>
		</match>
	</rule>
</rules>
