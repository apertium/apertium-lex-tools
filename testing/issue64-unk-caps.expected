^bassin<n><sg>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^mer<n><pl>/mar<n><sg>$
^bassin<n><sg>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^fleuve<n><pl>/flavor<n>$
^bassin<n><sg>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^London<np><top><sg>/London<np>$
^bassin<n><sg>/blah<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*mer/*mer$
^bassin<n><sg>/blah<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*M/*M$
^bassin<n><sg>/conca<n>$ ^de<pr>/a<pr>$ ^el<det><pl>/la<det><pl>$ ^*Mn/*Mn$
